airflow/models/baseoperator.py

```diff
@@ -907,7 +907,7 @@ def __init__(
             max_active_tis_per_dag = task_concurrency
         self.max_active_tis_per_dag: int | None = max_active_tis_per_dag
         self.max_active_tis_per_dagrun: int | None = max_active_tis_per_dagrun
-        self.do_xcom_push = do_xcom_push
+        self.do_xcom_push: bool = do_xcom_push
```

Comment:
Is this a Mypy issue? (Just asking since it doesn’t seem to be relevant to anything else)

Reply:
Correct